components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -208,6 +305,13 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:roles:{id}";
     option (chef.automate.api.iam.policy).action = "iam:roles:update";
   };
+  /*
+  Create a project
+
+  Creates a new project to be used in the IAM policies that control permissions in Automate.
+
+  A project defines the scope for a set of resources. A resource can be in more than one project.
```

Comment:
Not sure if we want to use `resources` here or not. Do we define resources anywhere else in the doc?
```suggestion
  A project defines the scope for a set of resources in a policy statement. A resource can be in more than one project.
```